samples/client/petstore/typescript-angular/package.json

```diff
@@ -16,7 +15,7 @@
     "angular": "^1.4.3"
   },
   "devDependencies": {
-    "tsd": "^0.6.3",
-    "typescript": "^1.5.3"
+    "typescript": "^2.1.4",
+    "@types/angular": "^1.5.21"
```

Comment:
@damienpontifex  If I understand correctly, the change is not backward compatible with the existing TS clients so please submit the PR against the `2.3.0` branch instead.